.github/workflows/cron_jerigon_zero_testing.yml

```diff
@@ -0,0 +1,106 @@
+name: Jerigon Zero Testing
+
+on:
+  # TODO - Change this before merge
+  # # Uncomment when ready to run on a schedule
+  # schedule:
+  #   # Run every Sunday at 12:00 AM (UTC)
+  #   - cron: "0 12 * * SUN"
+  push:
+    branches: [develop]
+  pull_request:
+    branches:
+      - "**"
+  workflow_dispatch:
+    branches:
+      - "**"
+
+env:
+  CARGO_TERM_COLOR: always
+  REGISTRY: ghcr.io
+
+jobs:
+  jerigon_zero_testing:
+    name: Jerigon Zero Testing - Integration and Benchmarking
+    runs-on: zero-reg
+    concurrency:
+      group: jerigon_zero_testing
+      cancel-in-progress: true
+    steps:
+      - name: Checkout zk_evm code
+        uses: actions/checkout@v4
+
+      - name: Setup Rust Toolchain
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+
+      - name: Set up Rust Cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          cache-on-failure: true
+
+      - name: Build the project
+        run: |
+          RUSTFLAGS='-C target-cpu=native -Zlinker-features=-lld -Copt-level=3' cargo build --release
+          sudo sysctl kernel.perf_event_paranoid=0
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Run Erigon Network
+        run: |
+          cd ..
+          # TODO - Download from IPFS
```

Comment:
TODOs need owners and issues please

Reply:
these todo are for me which i am working upon and dont forget
not a pending item

Reply:
I would like this commented with ownership and an issue in the case that a developer _does_ forget.
Simply search for `TODO` in the codebase and you'll find many :)